jest.config.ts

```diff
@@ -0,0 +1,192 @@
+/*
+ * For a detailed explanation regarding each configuration property and type check, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+export default {
+  // All imported modules in your tests should be mocked automatically
```

Comment:
Maybe we could get rid of the commented lines, if we need them later we can add them again.